events/wc_undead_events.txt

```diff
@@ -2737,4 +2737,172 @@ character_event = {
 			factor = 0
 		}
 	}
+}
+
+#############################
+#		Random events					#
+#############################
+### Random Necromantic events
+character_event = {
+	id = WCUND.9900
+
+	is_triggered_only = yes
+	hide_window = yes
+
+	min_age = 16
+	only_playable = yes
+	only_capable = yes
+	prisoner = no
+
+	trigger = {
+		is_alive = yes
+		is_dying = no
+
+		can_be_undead_trigger = yes
+	}
+
+	weight_multiplier = {
+		days = 1
+		modifier = {
+			factor = 2
+			OR = {
+				is_death_knight_class_trigger = yes
+				is_necromancer_class_trigger = yes
+				true_religion_group = necromancy_group
+			}
+		}
+		modifier = {
+			factor = 1.6
+			trait = cynical
+		}
+		modifier = {
+			factor = 1.6
+			trait = being_undead
+		}
+		modifier = {
+			factor = 1.2
+			trait = ambitious
+		}
+		modifier = {
+			factor = 1.2
+			is_mage_class_trigger = yes
+		}
+	}
+
+	immediate = {
+		random_list = {
+			10 = {
+				trigger = {
+					is_inaccessible_trigger = no
+					
+					NOT = { has_character_flag = lich_king_vision_happend_flag }	# Happens once in lifetime
+					
+					lich_king_vision_trigger = yes
+				}
+				set_character_flag = lich_king_vision_happend_flag
+				
+				character_event = { id = WCUND.9910 }	# Lich King vision
+			}
+			1 = {	# Small chance of nothing
+			}
```

Comment:
Event is already random, why do you need this superfluous thing?

Reply:
Purged in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1053/commits/bf15f659a7a70b482935703d35ebecedc9367beb.